golang/pkg/cli/display.go

```diff
@@ -0,0 +1,50 @@
+package cli
+
+import (
+	"os"
+	"path"
+
+	"github.com/rs/zerolog/log"
+)
+
+func PrintWelcomeMessage(settingsPath string) {
+	log.Info().Msgf("The config file is located at %s, where you can turn this message off.", settingsPath)
+	log.Info().Msgf("If you have any questions head to our Discord - https://discord.gg/pZWbd4fxga ! We're happy to help!")
+	log.Info().Msgf("You can learn more about the project at https://docs.dyrector.io, if you found this project useful please " +
+		"give us a star - https://github.com/dyrector-io/dyrectorio")
+}
+
+func PrintInfo(state *State) {
+	log.Warn().Msg("🦩🦩🦩 Use the CLI tool only for NON-PRODUCTION purpose. 🦩🦩🦩")
```

Comment:
I know you did not change this but `...NON-PRODUCTION purposes.` looks better, no?